### DISCUSSION TITLE
Does AWS S3 C++ SDK support lambda access point for GetObject?

### DISCUSSION THREAD

**Discussion by ruochenj123** [aws/aws-sdk-cpp#1908] [Q&A] [answered]
I have a lambda function for S3 bucket. I tested it using Python boto3 and it worked well. However, when I tried to use C++ SDK it gave an error of "curlCode: 7, Couldn't connect to server". I just copied ARN of my lambda access point and put it into GetObject API. Is there something wrong here or it is not supported?

**Comment by vudh1** [comment, marked as answer]
Are you using [this code sample](https://docs.aws.amazon.com/code-samples/latest/catalog/cpp-s3-get_object.cpp.html) for running?

**Reply by ruochenj123** [reply]
Yes. But I changed the bucket name to the lambda access point ARN (e.g., arn:aws:s3-object-lambda:us-east-1:....). I also tried only using access point without lambda function and it worked well.
